sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/output.py

```diff
@@ -97,6 +106,9 @@ def __init__(self, *, type=AssetTypes.URI_FOLDER, path=None, mode=None, descript
         self.mode = mode
         # use this field to determine the Output is control or not, currently hide in kwargs
         self.is_control = kwargs.pop("is_control", None)
+        self.min = kwargs.pop("min", None)
```

Comment:
as we synced in the meeting, we shall not have min,max,default defined for output

Reply:
ok, This PR can be discarded because there is no need to modify the code.